pkg/restapi/vault/operation/operations.go

```diff
@@ -47,15 +48,15 @@ func New(v vault.Vault) *Operation {
 
 // GetRESTHandlers get all controller API handler available for this service.
 func (o *Operation) GetRESTHandlers() []support.Handler {
-	return []support.Handler{
+	return append([]support.Handler{
 		support.NewHTTPHandler(CreateVaultPath, http.MethodPost, o.CreateVault),
 		support.NewHTTPHandler(DeleteVaultPath, http.MethodDelete, o.DeleteVault),
 		support.NewHTTPHandler(SaveDocPath, http.MethodPost, o.SaveDoc),
 		support.NewHTTPHandler(GetDocMetadataPath, http.MethodGet, o.GetDocMetadata),
 		support.NewHTTPHandler(CreateAuthorizationPath, http.MethodPost, o.CreateAuthorization),
 		support.NewHTTPHandler(GetAuthorizationPath, http.MethodGet, o.GetAuthorization),
 		support.NewHTTPHandler(DeleteAuthorizationPath, http.MethodDelete, o.DeleteAuthorization),
-	}
+	}, operation.New().GetRESTHandlers()...)
```

Comment:
Should not use here. Please use it in start.go like all other projects

Reply:
Done!